lib/SearchInput/index.js

```diff
@@ -32,19 +31,19 @@ class SearchInputField extends Component {
     const { placeholder } = this.props;
     const hasSearch = this.state.query ? 'is-visible' : '';
 
-    const iconClass = (() => `fa fa-search c-input-search__icon ${hasSearch}`)();
+    const iconClass = (() => `fa fa-search search-input__icon ${hasSearch}`)();
```

Comment:
Good shout. I've switched this round and done a little cleanup on the other const :)